micrometer-core/src/test/java/io/micrometer/core/instrument/simple/SimpleMeterRegistryTest.java

```diff
@@ -171,24 +171,16 @@ void stringRepresentationOfMetersShouldBeOk() {
                 )
         ).register(registry);
 
-        assertThat(registry.getMetersAsString()).isEqualTo(""
-                + "answers.count(COUNTER)[correct='true'] 2.0\n"
-                + "answers.count(COUNTER)[correct='false'] 1.0\n"
-                + "cache.latency.count.seconds(TIMER)[] 5.0\n"
-                + "cache.latency.total_time.seconds(TIMER)[] 0.1\n"
-                + "cache.miss.count(COUNTER)[] 42.0\n"
-                + "custom.meter.value(OTHER)[] 42.0\n"
-                + "custom.meter.unknown(OTHER)[] 21.0\n"
-                + "handler.active_tasks.seconds(LONG_TASK_TIMER)[] 1.0\n"
-                + "handler.duration.seconds(LONG_TASK_TIMER)[] 3.0\n"
-                + "latency.count.seconds(TIMER)[method='GET', service='test', uri='/api/people'] 10.0\n"
-                + "latency.total_time.seconds(TIMER)[method='GET', service='test', uri='/api/people'] 0.29\n"
-                + "latency.max.seconds(TIMER)[method='GET', service='test', uri='/api/people'] 0.038\n"
-                + "processing.time.value.seconds(GAUGE)[] 0.3\n"
-                + "request.size.count.bytes(DISTRIBUTION_SUMMARY)[] 10.0\n"
-                + "request.size.total.bytes(DISTRIBUTION_SUMMARY)[] 1450.0\n"
-                + "request.size.max.bytes(DISTRIBUTION_SUMMARY)[] 190.0\n"
-                + "temperature.value.celsius(GAUGE)[] 24.0");
+        assertThat(registry.getMetersAsString()).isEqualTo("answers(COUNTER)[correct='true']; count=2.0\n" +
+                "answers(COUNTER)[correct='false']; count=1.0\n" +
+                "cache.latency(TIMER)[]; count=5.0, total_time=0.1 seconds\n" +
```

Comment:
I was wondering why this didn't have a max, and it's because it is a `FunctionTimer`. This brings up the question of do we want that level of information available in the output or not. We don't have to answer that for this pull request, but it's something to consider.